server/claim-review/claim-review.service.ts

```diff
@@ -36,89 +35,69 @@ export class ClaimReviewService {
     ) {}
 
     async listAll(page, pageSize, order, query, latest = false) {
-        const aggregation = [];
-
-        aggregation.push(
-            {
-                $sort: latest
-                    ? { date: -1 }
-                    : { _id: order === "asc" ? 1 : -1 },
-            },
-            { $match: query },
-            lookUpPersonalityties(TargetModel.ClaimReview),
-            lookupClaims(TargetModel.ClaimReview),
-            { $unwind: "$claim" },
-            lookupClaimRevisions(TargetModel.ClaimReview),
-            { $unwind: "$claim.latestRevision" },
-            {
-                $match: {
-                    "claim.isHidden": query.isHidden,
-                    "claim.isDeleted": false,
-                    "claim.nameSpace":
+        const pipeline = this.ClaimReviewModel.find(query)
```

Comment:
follow-up: we need to add namespace to claim review to avoid bad populate results